### PR TITLE
add undo bar to the reset implementation and add Toast when trying to reset empty selection

### PR DESCRIPTION
**Pull request by sapirshp**

### CODE REVIEW

File: app/src/main/java/com/example/meetapp/MenuHandler.java [hunk 1]
```diff
@@ -48,16 +47,36 @@ void handleGroupDetails(Context context)
...
+                System.out.println(selectedSlot.getDate());
+            }
+            for (TimeSlot slotToReset : slotsToReset) {
+                calendarSlotsHandler.clickedOff(slotToReset);
             }
+            new UndoBarController.UndoBar(activity).message("Reset Selection").style(UndoBarController.UNDOSTYLE).listener(new UndoBarController.AdvancedUndoListener() {
+                @Override
+                public void onHide(@Nullable Parcelable token) {
+
+                }
```
Comment (Chen-Ben-yashar) [inline app/src/main/java/com/example/meetapp/MenuHandler.java]: empty line.
for empty functions, use { }  in same line.
(I assume the functions are necessary, if not - remove as well)

File: app/src/main/java/com/example/meetapp/MenuHandler.java [hunk 2]
```diff
@@ -48,16 +47,36 @@ void handleGroupDetails(Context context)
...
+
+                @Override
+                public void onClear(@NonNull Parcelable[] token) {
+
```
Comment (Chen-Ben-yashar) [inline app/src/main/java/com/example/meetapp/MenuHandler.java]: same as above

File: app/src/main/java/com/example/meetapp/MenuHandler.java [hunk 3]
```diff
@@ -48,16 +47,36 @@ void handleGroupDetails(Context context)
...
-                resetFunction.accept(slotToReset);
+    void handleResetTimeChoice(Activity activity, final CalendarSlotsHandler calendarSlotsHandler) {
+        slotsToReset.clear();
+        if (calendarSlotsHandler.getSlotSelections().isEmpty()){
+            Toast.makeText(activity,activity.getString(R.string.resetEmptySelection), Toast.LENGTH_LONG).show();
+        }else {
+            for (TimeSlot selectedSlot : calendarSlotsHandler.getSlotSelections().keySet()) {
+                slotsToReset.add(selectedSlot);
+                System.out.println(selectedSlot.getDate());
+            }
```
Comment (Chen-Ben-yashar) [inline app/src/main/java/com/example/meetapp/MenuHandler.java]: I would add a coment explaining why the 2 for loops are needed, but it is up to you

Comment (sapirshp) [reply app/src/main/java/com/example/meetapp/MenuHandler.java]: don't think its necessary

File: app/src/main/java/com/example/meetapp/MenuHandler.java (hunk 2 above)
Comment (sapirshp) [reply app/src/main/java/com/example/meetapp/MenuHandler.java]: fixed

File: app/src/main/java/com/example/meetapp/MenuHandler.java (hunk 1 above)
Comment (sapirshp) [reply app/src/main/java/com/example/meetapp/MenuHandler.java]: fixed

**Pull request merged**